lambdas/build-cri-oauth-request/src/main/java/uk/gov/di/ipv/core/buildcrioauthrequest/BuildCriOauthRequestHandler.java

```diff
@@ -371,11 +369,12 @@ private JWEObject signEncryptJar(
     private EvidenceRequest getEvidenceRequestForF2F(
             List<VerifiableCredential> vcs, Vot requestedVot) {
         var gpg45Scores = gpg45ProfileEvaluator.buildScore(vcs);
-        List<Gpg45Scores> requiredEvidences =
+        var isFraudCheckUnavailable = VcHelper.isFraudCheckUnavailable(vcs);
+        var requiredEvidences =
                 gpg45Scores.calculateGpg45ScoresRequiredToMeetAProfile(
-                        requestedVot.getSupportedGpg45Profiles());
+                        requestedVot.getSupportedGpg45Profiles(isFraudCheckUnavailable));
```

Comment:
This won't do anything for now (because F2F can never meet M1C), but it might be useful if we introduce L1B